### PR TITLE
include docs on how to use outputs

### PR DESCRIPTION
**Pull request by drewmullen**
Closes: #105

### CODE REVIEW

File: examples/public_private_flow_logs/main.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@ data "aws_availability_zones" "current" {}
 
 module "vpc" {
   # source  = "aws-ia/vpc/aws"
-  # version = ">= 3.0.2"
+  # version = ">= 4.0."
```
Comment (tlindsay42) [inline examples/public_private_flow_logs/main.tf:5]: Minor nit: patch version is missing

**Review by tlindsay42 — APPROVED** [review]
lgtm w/ 1 minor nit

**Comment by drewmullen** [comment]
thank you for making that update @pablo19sc 

@tlindsay42 this is ready for re-review when you have a second :)

File: examples/public_private_flow_logs/main.tf [hunk 2]
```diff
@@ -1,9 +1,8 @@
 data "aws_availability_zones" "current" {}
 
 module "vpc" {
-  # source  = "aws-ia/vpc/aws"
-  # version = ">= 3.0.2"
-  source = "../.."
+  source  = "aws-ia/vpc/aws"
+  version = ">= 4.2.0"
```
Comment (tlindsay42) [inline examples/public_private_flow_logs/main.tf:5]: ❤️

**Review by tlindsay42 — APPROVED** [review]

**Pull request merged**